Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,261 @@
+
+A: Definitely not! Here are the [selection criteria](#selection-criteria) we use when selecting contributors for GSoC. Note that tenure is explicitly not part of these criteria.
+
+**Q: What are the minimum number of PRs that one should have?**
+
+A: You should have at least 2 PRs merged, but they don't need to be large. Beyond that, remember that quality is more important than quantity. It is better to submit a non-trivial PR rather than a one-line wording change. Start with starter issues, then prioritize ones that touch the area(s) of the codebase which are related to your project.
+
+**Q: What is the total number of contributors that will be accepted?**
+
+A: As many as we think will succeed, though the Google GSoC admins may impose limits based on how they decide to distribute contributor slots among the different open-source organizations.
+
+**Q: I do not have any experience in skill XYZ. Is some certification required?**
+
+A: Try to work on good first issues and take courses online. In the field of software development, it is common to develop experience and expertise as you take up and complete projects successfully. We do not require any formal certification of particular skills.
+
+**Q: Is it okay if I only focus on the frontend or backend?**
+
+A: This probably depends on the project(s) you wish to apply for. However, note that the ability to be effective in both the frontend and backend will open up more opportunities for you, since projects often touch multiple layers of the stack.
+
+**Q: The [Google GSoC FAQ](https://developers.google.com/open-source/gsoc/faq#can_someone_already_participating_in_open_source_be_a_gsoc_contributor) mentions that the program is only for new contributors. I have already contributed to Oppia and I have write access. Can I still participate?**
+
+A: The GSoC program is open to new and beginner contributors to open source. Some of our contributors with write access are still beginner contributors, whereas some of our other contributors with write access will not qualify because they are experienced contributors. If you have only recently received write access, or have been contributing to Oppia for less than a year, you are probably still a beginner contributor. If the previous sentence does not apply to you, and you want to know which group you fall into, please contact **@vojtechjelinek** or **@seanlip** for a decision.
+
+## Dates and Deadlines
+
+Noteworthy dates for 2023 ([Full Timeline](https://developers.google.com/open-source/gsoc/timeline)):
+
+- **Jan 23 - Feb 7**: Mentoring organizations apply
+- **Feb 22**: Mentoring organizations are announced
+- **Mar 20 - Apr 4**: GSoC contributor application period
+- **May 4**: Accepted GSoC contributors are announced
+- **May 4 - May 28**: Community bonding period
+- **May 29 - Nov 6**: GSoC contributors enjoy the summer by contributing code to their projects
+- **Nov 13**: GSoC officially ends
```

Comment:
We aren't allowing projects to run this late, right?

Reply:
Maybe here we can do something like (replacing the durations with the exact dates):

- First 8 weeks: Standard coding period for Milestone 1
- Next 8 weeks: Standard coding period for Milestone 2

and add a note explaining that the coding period dates are strict and intended to balance offering flexibility to contributors with the need on the org admin side to have a streamlined process and a more time-bounded GSoC period.

(We can use the proposal template to explain the finer details of the external evals etc. but I think  the above might be the simplest way to communicate our plans for the timeline.)

Reply:
I have corrected the dates according to the timeline sheet shared by @U8NWXD